aspnetcore/blazor/data-binding.md

```diff
@@ -41,10 +41,10 @@ Using `@bind` with the `CurrentValue` property (`<input @bind="CurrentValue" />`
 
 When the component is rendered, the `value` of the input element comes from the `CurrentValue` property. When the user types in the text box and changes element focus, the `onchange` event is fired and the `CurrentValue` property is set to the changed value. In reality, the code generation is more complex because `@bind` handles cases where type conversions are performed. In principle, `@bind` associates the current value of an expression with a `value` attribute and handles changes using the registered handler.
 
-In addition to handling `onchange` events with `@bind` syntax, a property or field can be bound using other events by specifying an [`@bind-value`](xref:mvc/views/razor#bind) attribute with an `event` parameter ([`@bind-value:event`](xref:mvc/views/razor#bind)). The following example binds the `CurrentValue` property for the `oninput` event:
+In addition to handling `onchange` events with `@bind` syntax, a property or field can be bound for other events by also including an `@bind:event` attribute with an `event` parameter. The following example binds the `CurrentValue` property for the `oninput` event:
```

Comment:
not sure whether this should be `for` or `on`?
```suggestion
In addition to handling `onchange` events with `@bind` syntax, a property or field can be bound on other events by also including an `@bind:event` attribute with an `event` parameter. The following example binds the `CurrentValue` property on the `oninput` event:
```

Reply:
I went with "for" here for a couple of reasons, but I don't think either is a bad choice. One was to imply **_purpose_**, which "for" does well. "On" seems like a good choice, too, because it implies location/time. Also in the 2nd case, we end up with "on" "onevent," which sounded a little repetitive.

Let's try another version of this that might be a further improvement of the first sentence. I'll try *imperative tone* (an order to the reader) on the next commit. I'm cool with "on" for the second sentence. I like that better this morning! :smile: